docs/source/workflows/about/sequential-executor.md

```diff
@@ -15,41 +15,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Sequential Executor
-
-A sequential executor is a control flow component that chains multiple functions together, where each function's output becomes the input for the next function. This creates a linear tool execution pipeline that executes functions in a predetermined sequence without requiring LLMs or agents for orchestration.
-
-## Features
-- **Sequential Function Chaining**: Chain multiple functions together where each function's output becomes the input for the next function
-- **Type Compatibility Checking**: Optionally validate that the output type of one function is compatible with the input type of the next function in the chain
-- **Streaming Support**: Configure individual functions to use streaming or non-streaming execution modes
-- **Error Handling**: Handle errors gracefully throughout the sequential execution process
-- **No LLM Required**: Execute deterministic workflows without the overhead of LLM calls
-- **Configurable Workflows**: Fully configurable via YAML for flexibility and productivity
-
----
+# Configure With the Sequential Executor
+A sequential executor is a deterministic workflow orchestrator that executes functions in a predefined linear order. This sections explores ways you can configure using the sequential executor.
 
 ## Requirements
 The sequential executor is part of the core NeMo Agent toolkit and does not require additional plugin installations.
 
-If you have performed a source code checkout, install this with the following command:
+To install NeMo Agent toolkit:
 
 ```bash
 uv pip install -e .
 ```
 
-If you have installed the NeMo Agent toolkit from a package, you can install this with the following command:
-
-```bash
-uv pip install "nvidia-nat"
-```
-
```

Comment:
It is important to retain the installation specific instructions here, as `uv pip install -e .` will fail for users who did not perform a source checkout.